packages/lesswrong/components/ea-forum/TargetedJobAd.tsx

```diff
@@ -504,76 +334,71 @@ const TargetedJobAd = ({ad, onDismiss, onExpand, onApply, onRemindMe, classes}:
+        <Tooltip title="Dismiss">
+          <Button className={classes.closeButton} onClick={onDismiss}>
+            <CloseIcon className={classes.closeIcon} />
+          </Button>
+        </Tooltip>
       </div>
+
       <div className={classes.mainRow}>
         <img src={adData.logo} className={classes.logo} />
         <div className={classes.bodyCol}>
-          <div className={classes.headerRow}>
-            <ForumIcon icon="Pin" className={classes.pinIcon} />
-            <h2 className={classes.header}>
-              <InteractionWrapper className={classes.inline}>
+          
+          <div>
+            <div className={classes.headerRow}>
+              <ForumIcon icon="Pin" className={classes.pinIcon} />
+              <h2 className={classes.header}>
                 <Link to={adData.bitlyLink} target="_blank" rel="noopener noreferrer">
                   {adData.role}
-                </Link>
-              </InteractionWrapper> at{adData.insertThe ? ' the ' : ' '}
-              <InteractionWrapper className={classes.inline}>
+                </Link> at{adData.insertThe ? ' the ' : ' '}
                 <HoverPreviewLink href={adData.orgLink}>
                   {adData.org}
                 </HoverPreviewLink>
-              </InteractionWrapper>
-            </h2>
-          </div>
-          <div className={classes.metadataRow}>
-            {adData.salary && <>
-              <div className={classes.metadata}>
-                {adData.salary}
-              </div>
-              <div>·</div>
-            </>}
-            <div className={classes.metadata}>
-              {adData.location}
+              </h2>
             </div>
-            {adData.roleType && <>
-              <div>·</div>
+            <div className={classes.metadataRow}>
+              {adData.salary && <>
+                <div className={classes.metadata}>
+                  {adData.salary}
+                </div>
+                <div>·</div>
+              </>}
               <div className={classes.metadata}>
-                {adData.roleType}
+                {adData.location}
               </div>
-            </>}
-            {adData.deadline && <>
-              <div>·</div>
-              <div className={classes.metadata}>
-                Deadline: {adData.deadline.format('MMM Do')}
-              </div>
-            </>}
+              {adData.roleType && <>
+                <div>·</div>
+                <div className={classes.metadata}>
+                  {adData.roleType}
+                </div>
+              </>}
+              {adData.deadline && <>
+                <div>·</div>
+                <div className={classes.metadata}>
+                  Deadline: {adData.deadline.format('MMM Do')}
+                </div>
+                {showRemindMe && <>
+                  <div>·</div>
+                  <button onClick={handleRemindMe} className={classes.reminderBtn}>
+                    Remind me
+                  </button>
```

Comment:
I think clicking this shouldn't close the ad, I can imagine clicking "Remind me" as a backup but then also clicking through to read more/apply immediately

Reply:
Although this wasn't introduced in this PR, so maybe it's been covered elsewhere

Reply:
Same comment for the apply button actually

Reply:
I mostly agree with you but I worry that the ads take up valuable space so I wanted to minimize how much attention they ask for. I think surprisingly few users dismiss the ad. I'll think about this more.

Reply:
Ok now that we changed the CTA to be "View job details" instead of "Apply" I think it makes more sense for it not to close. I lean towards removing the "close on click" functionality but still no longer showing the ad after you leave the page and return (like if you refresh the page). How does that sound to you?

Reply:
Ah I realized that I actually need to track the reminder separately if I do this, so I pushed a new commit with that (plus I added one more job). I'd appreciate you taking a quick look but it's ok if you don't have time - either way I'll prob aim to deploy this branch my Monday EOD.